app/controllers/admin/batches_controller.rb

```diff
@@ -30,7 +30,7 @@ def show
     respond_to do |format|
       format.html
       format.pdf do
-        render pdf: pdf_filename, page_size: 'A4', layout: 'pdf', margin: { top: 0, bottom: 0, left: 0, right: 0 }, dpi: '300'
+        render pdf: pdf_filename, page_size: 'A4', layout: 'pdf', margin: { top: 0, bottom: 0, left: 0, right: 0 }, dpi: '300', show_as_html: params[:debug]
```

Comment:
:elephant: whats this `[:debug]`

Reply:
@mischa-s That tells `wicked_pdf` to convert the pdf to html for the tests!! Cool, hey?